tsuru/client/services.go

```diff
@@ -688,15 +687,14 @@ func (c *ServiceInstanceRemove) Run(ctx *cmd.Context, client *cmd.Client) error
 	if err != nil {
 		return err
 	}
-	var jsonMsg tsuruIo.SimpleJsonMessage
-	json.Unmarshal(result, &jsonMsg)
+	jsonMsg, jsonErr := getErrorAndMessageInSimpleJSON(strings.Split(string(result), "\n"))
 	var msgError error
-	if jsonMsg.Error != "" {
-		msgError = errors.New(jsonMsg.Error)
+	if jsonErr != "" {
+		msgError = errors.New(jsonErr)
```

Comment:
Why don't you return this error directly in `getErrorAndMessageInSimpleJSON` as the second parameter?